README.md

```diff
@@ -1,29 +1,75 @@
 # InstaTonne
-distributed social media app for CMPUT404 :)
+[InstaTonne](https://cmput404-group6-instatonne.herokuapp.com/app/) is a revolutionary social networking app that embraces the interconnectedness and the web's peer-to-peer nature. Our aim is to provide users with more autonomy and choice while connecting them to various independent social networks without being limited to a single platform like Facebook, Instagram or Twitter.

```

Comment:
maybe change to this:
that embraces the interconnectedness and peer-to-peer nature of the web.

Reply:
kk